calculated_policies/aws_lambda_function_approved_usage/README.md

```diff
@@ -0,0 +1,110 @@
+# AWS Lambda - Approve a Lambda function with an existense of tag
+
+## Use case
+
+The business owner of the AWS environment wants to approve only the Lambda functions that are with tag associated to it.
+
+## Implementation details
+
+This Terraform template creates a smart folder and applies calculated policies on the policies:
+
+- `AWS > Lambda > Function > Approved`
+- `AWS > Lambda > Function > Approved > Usage`
+
+The Calculated policy checks the Lambda metadata for existence of ApplicationID tag and if the tag doesn't exist and it is older than five minutes it will return "Not approve" other wise it will approve. This gives users five minutes to tag their resources or have them deleted.
+
+### Template input (GraphQL)
+
+The template input to a calculated policy is a GraphQL query.
+
+In this case the query finds the Lambda function's attribute in the Lambda metadata.
+
+```graphql
+{
+  resource {
+    data
+    metadata
+    trunk {
+      title
+    }
+    turbot {
+      akas
+      id
+      tags
+    }
+  }
+}
+```
+
+### Template (Nunjucks)
+
+```nunjucks
+{%- set result = "Approved" -%}
+{%- set current_time = now | date("constructor") | date("getTime") -%}
+{%- set lastmodified_time = $.resource.data.Configuration.LastModified | date("getTime") -%}
+{%- if "ApplicationID" not in $.resource.turbot.tags and duration > 300000 -%}
+  {%- set result = "Not approved" -%}
+{%- endif -%}
+{{ result }}
+```
+
+The template itself is a [Nunjucks formatted template](https://mozilla.github.io/nunjucks/templating.html).
+
+## Prerequisites
+
+To run Turbot Calculated Policies, you must install:
+
+- [Terraform](https://www.terraform.io) Version 12
```

Comment:
needs to be 13

Reply:
Required terraform version has been updated as 13